### DISCUSSION TITLE
GigaPixel scene - no discernable effect

### DISCUSSION THREAD

**Discussion by fieldWorks-3D** [NVlabs/instant-ngp#250] [Q&A] [answered]
Thank you for making this code available.
I was using the Einstein sample scene, and although my fan goes crazy with training, I see no increased, generated resolution in the zoomable image. I'm not sure what to expect, though the pixel density appears to be the same as the original when compared side-by-side.
(Also the README mentions the Image2bin script for using with other photos, but that is not in the repository.)
Is it possible to upscale an image and save the upscaled version using this GUI?

**Comment by Tom94** [comment, marked as answer]
Hi there, this codebase doesn't do superresolution. It just fits the provided image -- think of it more as compression / representation.
